test/Pester/EditorServices.Integration.Tests.ps1

```diff
@@ -464,6 +464,22 @@ Get-Foo
         $edit.Arguments.EndColumnNumber | Should -Be 4
     }
 
+    It "handles textDocument/completion properly" {
```

Comment:
There's plenty of opportunities here to add more completion tests.  :-)  Thinking of parameter names, paths with spaces, variable names, etc.

Reply:
For sure! These e2e tests will act as smoke tests. After everything is ported over, I add tests for every handler as unit tests.